### ISSUE TITLE
unable to start Dota on Vulkan

### ISSUE THREAD

**Issue by krav559108** [ValveSoftware/Dota-2#2193]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): 
* Have you checked for system updates?: [Yes/No]
* Are you using the latest stable video driver available for your system? [Yes/No]
* Have you verified the game files?: [Yes/No]

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large pastes as a [Github Gist](https://gist.github.com/).

#### Steps for reproducing this issue:

Computer information:
    Manufacturer: Apple
    Model: MacBook Air10.1
    Type: Notebook
    Touch input is not supported

CPU:
    Processor manufacturer: VirtualApple
    Processor manufacturer: Apple M1
    Processor family: 0x6
    Processor Model: 0xc
    Processor stepping: 0x0
    Processor type: 0x0
    Clock frequency: 2400 MHz
    Logical processors: 8
    Physical processors: 8
    Hyper Threading: Not Supported
    FCMOV: Not supported
    SSE2: Supported
    SSE3: Supported
    SSSE3: Supported
    SSE4a: Not supported
    SSE41: Supported
    SSE42: Supported
    AES: Supported
    AVX: Not supported
    AVX2: Not supported
    AVX512F: Not supported
    AVX512PF: Not supported
    AVX512ER: Not supported
    AVX512CD: Not supported
    AVX512VNNI: Not supported
    SHA: Not supported
    CMPXCHG16B: Supported
    LAHF/SAHF: Supported
    PrefetchW: Not supported

OS version:
    macOS 13.0.0 (64 bit)

Video card:
    Model: Apple M1
    Driver version: 227.2.40
    OpenGL version: 4.1
    Color depth: 32 bits/pixel
    Refresh rate: 60Hz
    Vendor ID: 0x106b
    Card ID: 0x0
    Version unknown
    Number of screens: 1
    Number of logical video cards: 1
    Main resolution Screen: 1440 x 900
    Desktop resolution: 1440 x 900
    Main size Screen: 11.26" x 7.05" (13.27" diag)
 28.6cm x 17.9cm (33.7cm diag)
    Main video memory: 8192 MB

Memory:
    RAM: 8192 MB

VR Devices:
    VR Headset: Not Detected

Miscellaneous:
    Interface language: Russian
    Media type: DVD
    Disk space: 233752 MB
    Most free disk: 42748 MB

Disk space:
    Number of SSDs: 0
    Number of hard drives: 0

1. y
2. y
3. y

Basically after update on 4.11.2022 game is not able to run on macbook air m1 (macos: 13.0 (22A380)) with vulkan render, its not starting also with - safe_mode so im forced to use -g| launch option and then it keeps crashing every time or there is no hero textures in the game. Before this update comes up everything was just fine. I had O issues with this game until this point. And i have friends who also uses mac for gaming and since yesterday they have same issues. So is there solution for this problem or i should wait for an update specifically for mac users? And i tried also reinstalling and checking if some files get corrupted or idk. So hope to hear soon from you.

**Comment by kisak-valve** [comment]
Hello @krav559108, this issue is already being tracked at #2190. Closing as a duplicate.
